docs/messenger/messenger.rst

```diff
@@ -0,0 +1,131 @@
+Messenger
+=========
+
+Messenger is a feature in Mautic 5.x that offers several significant benefits:
+
+.. _highlights:
+
+**Load distribution/control**
+    It enables you to balance and control the server load effectively.
+
+**Improved client response**
+    By processing requests in the background, it ensures that the client receives a response before the completion of the request processing.
+
+**Scalability**
+    It provides the flexibility to scale your Mautic instance by adding more workers.
```

Comment:
Will marketers know what workers are? I don't know if we have explained that concept at all before.